.github/workflows/test-and-deploy.yml

```diff
@@ -53,7 +55,7 @@ jobs:
   deploy:
     if: github.repository_owner == 'compiler-explorer' && github.event_name == 'push'
     needs: [test, build_dist]
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
ditto. We can investigate our options here and do a test to our staging environment. Can you make a separate PR with these changes, please?